api/package.json

```diff
@@ -109,7 +110,9 @@
     "db:prepare": "npm run db:delete && npm run db:create && npm run db:migrate",
     "db:seed": "knex --knexfile db/knexfile.js seed:run",
     "db:reset": "npm run db:prepare && npm run db:seed",
-    "lint": "eslint .",
+    "lint": "npm run lint:file-and-directory-name && npm run lint:code",
+    "lint:file-and-directory-name": "npx @ls-lint/ls-lint",
```

Comment:
t'es obligé de passer par npx ici ? je pensais que dans le contexte de npm il était déjà dans les node-modules

Reply:
Npx exécute depuis les `node_modules` : 
> Executes <command> either from a local node_modules/.bin, or from a central cache, installing any packages needed in order for <command> to run. 
>
> By default, npx will check whether <command> exists in $PATH, or in the local project binaries, and execute that. If <command> is not found, it will be installed prior to execution.

Reply:
Bien vu, c'est corrigé